### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by coliff**
Bump version number to 1.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ back and forth between your real source code and the browser view.
 * **Do it yourself.** Because Brackets is open source, and built with HTML, CSS
 and JavaScript, you can [help build](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md) the best code editor for the web.
 
-Brackets is at 1.0 and we're not stopping there. We have many feature ideas on our
+Brackets is at 1.2 and we're not stopping there. We have many feature ideas on our
```
Comment (redmunds) [inline README.md]: I don't think we want to update this for every release. I think the intent is more like "Brackets is at version 1 and we're not stopping there."

Comment (le717) [inline README.md]: How about "Brackets may have reached version 1, but we're not stopping there."?

Comment (peterflynn) [inline README.md]: @le717 Nice, I like that wording

Comment (coliff) [inline README.md]: LGTM :+1:

**Comment by nethip** [comment]
This not only looks good to but inspiring as well :smile:

**Comment by peterflynn** [comment]
@nethip I think there was a misunderstanding -- in the thread above we were talking about using a _different_ wording from what's in this PR, so we don't have to update the README every release.  So we wanted to wait to merge this until the wording was updated...  @coliff can you put up a new PR with the new wording?

**Comment by coliff** [comment]
@peterflynn Sure - are we going with:
'Brackets may have reached version 1, but we're not stopping there.'
Please confirm and I'll open a PR.

**Comment by nethip** [comment]
@peterflynn @coliff My Bad I don't know why I was under the impression that the suggested wording was already in the PR. I will review the new PR going to be uploaded by @coliff

**Comment by redmunds** [comment]
@coliff 

> are we going with: 'Brackets may have reached version 1, but we're not stopping there.' ?
> Please confirm and I'll open a PR

Yes

**Pull request merged**